synapse/tests/test_model_inet.py

```diff
@@ -1295,3 +1295,44 @@ def addrows(mesg):
                 self.len(2, core.getRowsByProp('syn:tagform:form', 'inet:web:logon'))
                 self.len(1, core.getRowsByProp('_:*inet:web:logon#hehe.hoho'))
                 self.len(1, core.getRowsByProp('_:*inet:web:logon#hehe'))
+
+    def test_model_inet_201710111553(self):
+
+        adds = []
+
+        iden, tick = guid(), now()
+        adds.extend([
+            (iden, 'tufo:form', 'inet:web:acct', tick),
+            (iden, 'inet:web:acct', 'vertex.link/pennywise1', tick),
+            (iden, 'inet:web:acct:site', 'vertex.link', tick),
+            (iden, 'inet:web:acct:user', 'pennywise', tick),
+            (iden, 'inet:web:acct:occupation', 'EnterTainEr', tick),
```

Comment:
I love this test btw.  I hear he has good balloon animals :balloon: :clown_face: